migration/migrator/data/course_tables.sql

```diff
@@ -2564,10 +2564,10 @@ CREATE TRIGGER gradeable_version_change AFTER INSERT OR DELETE OR UPDATE ON publ
 
 
 --
--- Name: late_day_exceptions late_day_extension_change; Type: TRIGGER; Schema: public; Owner: -
+-- Name: excused_absence_extensions late_day_extension_change; Type: TRIGGER; Schema: public; Owner: -
 --
 
-CREATE TRIGGER late_day_extension_change AFTER INSERT OR DELETE OR UPDATE ON public.late_day_exceptions FOR EACH ROW EXECUTE PROCEDURE public.late_day_extension_change();
+CREATE TRIGGER late_day_extension_change AFTER INSERT OR DELETE OR UPDATE ON public.excused_absence_extensions FOR EACH ROW EXECUTE PROCEDURE public.late_day_extension_change();
```

Comment:
I didn't want to change it yet because I wanted to keep the PR short, so I just only wanted to change to the variables that had to deal with the name of the table and column. After this PR is merged, I would then continue to change other variable names and create other PRs for it.